app/demo-loop/simple-driver.py

```diff
@@ -57,6 +57,8 @@
 
 print("Received {} bytes of data".format(len(result.stdout)))
 out_text = result.stdout.decode()
+# Filter out spurious HepMC3 output
+out_text = out_text[out_text.find('\n{') + 1:]
```

Comment:
This might cause newer HepMC3 versions (which don't write to stdout) to fail...

Reply:
I did think about that case too -- in the case where the string `\n{` is not in the output (as happens when HepMC3 doesn't spit anything out), `out.text.find(...)` returns -1, so it ends up being `out_text[0:]`.

Reply:
Sneaky!